MAUI/DataForm/getting-started.md

```diff
@@ -740,3 +740,7 @@ public class DataFormItemManagerEditorExt : DataFormItemManager
 
 {% endhighlight %}
 {% endtabs %}
+
+You can find the complete getting started sample from this [link](https://help.syncfusion.com/maui/dataform/getting-started).
+
+N> You can refer to our [.NET MAUI DataForm](https://www.syncfusion.com/maui-controls/maui-dataform) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI DataForm example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/DataForm) that shows you how to render and configure the Kanban Board.
```

Comment:
Check on this
![image](https://github.com/user-attachments/assets/10252cb9-3244-41ae-8750-8c700f359c8e)